libres/lib/analysis/stepwise.cpp

```diff
@@ -8,370 +8,356 @@
+                            //prediction_error += e_estimated_value*e_estimated_value;
+                        }
+                    }
+                    matrix_free(x_vector);
+                }
             }
-
-          }
-          matrix_free( x_vector );
         }
-      }
-    }
 
-    free( randperms );
-    bool_vector_free( active_rows );
-  }
+        free(randperms);
+        bool_vector_free(active_rows);
+    }
 
-  /*inactivate the test_var-variable after completion*/
-  bool_vector_iset( stepwise->active_set , test_var , false );
-  return prediction_error;
+    /*inactivate the test_var-variable after completion*/
+    bool_vector_iset(stepwise->active_set, test_var, false);
+    return prediction_error;
 }
 
+void stepwise_estimate(stepwise_type *stepwise, double deltaR2_limit,
+                       int CV_blocks) {
+    int nvar = matrix_get_columns(stepwise->X0);
+    int nsample = matrix_get_rows(stepwise->X0);
+    double currentR2 = -1;
+    bool_vector_type *active_rows = bool_vector_alloc(nsample, true);
 
-void stepwise_estimate( stepwise_type * stepwise , double deltaR2_limit , int CV_blocks) {
-  int nvar          = matrix_get_columns( stepwise->X0 );
-  int nsample       = matrix_get_rows( stepwise->X0 );
-  double currentR2 = -1;
-  bool_vector_type * active_rows = bool_vector_alloc( nsample , true );
-
-
-  /*Reset beta*/
-  for (int i = 0; i < nvar; i++) {
-    matrix_iset(stepwise->beta, i , 0 , 0.0);
-  }
-
-
+    /*Reset beta*/
+    for (int i = 0; i < nvar; i++) {
+        matrix_iset(stepwise->beta, i, 0, 0.0);
+    }
 
-  bool_vector_set_all( stepwise->active_set , false );
+    bool_vector_set_all(stepwise->active_set, false);
 
-  double MSE_min = 10000000;
-  double Prev_MSE_min = MSE_min;
-  double minR2    = -1;
+    double MSE_min = 10000000;
+    double Prev_MSE_min = MSE_min;
+    double minR2 = -1;
 
-  while (true) {
-    int    best_var = 0;
-    Prev_MSE_min = MSE_min;
+    while (true) {
+        int best_var = 0;
+        Prev_MSE_min = MSE_min;
 
-    /*
+        /*
       Go through all the inactive variables, and calculate the
       resulting prediction error IF this particular variable is added;
       keep track of the variable which gives the lowest prediction error.
     */
-    for (int ivar = 0; ivar < nvar; ivar++) {
-      if (!bool_vector_iget( stepwise->active_set , ivar)) {
-        double newR2 = stepwise_test_var(stepwise , ivar , CV_blocks);
-        if ((minR2 < 0) || (newR2 < minR2)) {
-          minR2 = newR2;
-          best_var = ivar;
+        for (int ivar = 0; ivar < nvar; ivar++) {
+            if (!bool_vector_iget(stepwise->active_set, ivar)) {
+                double newR2 = stepwise_test_var(stepwise, ivar, CV_blocks);
+                if ((minR2 < 0) || (newR2 < minR2)) {
+                    minR2 = newR2;
+                    best_var = ivar;
+                }
+            }
         }
-      }
-    }
 
-    /*
+        /*
```

Comment:
The indentation of this comment looks a bit funky since indentation of the code has changed.